requirements.in

```diff
@@ -31,7 +31,7 @@ mysql-connector-python==8.0.26
 docker==4.4.4  # can't use >=5 because of tcconfig==0.26.0
 python-jenkins==1.7.0
 ssh2-python==1.0.0
-argus-alm==0.8.1
+argus-alm==0.10.0
```

Comment:
we must change version for this switch ?
since backporting this would be a bit problematic, since we'll need to rebuild hydra image for each branch (since there are other changes to hydra image which wasn't backported, and we don't know if not having them would break SCT or not)

Reply:
I think 0.8.1 might be missing some fixes and improvements to the client, so might be annoying to not update

Reply:
if we don't have to have those improvement for release branches, I'm would rather not touch them.

For now let do this change on master, and we'll wait for a week before considering backports

Reply:
Okay